### DISCUSSION TITLE
❔ Is there a way to Name Examples

### DISCUSSION THREAD

**Discussion by toddgeist** [RobinTail/express-zod-api#962] [Ideas]
I have examples on input methods working, But, they are just named "example1", "example2", etc.  I don't see a way to use `withMeta(schema).examples()` to name those examples.  Am I missing something?

Thanks for a great library :-)

**Comment by RobinTail** [comment]
Hello @toddgeist ,

unfortunately naming of examples is not supported yet.
I didn't expect that it could be useful.
Perhaps I'll make it in the future... maybe along with descriptions.

**Reply by toddgeist** [reply]
You are right. It isn't that useful.  Just a nice to have.

Thanks for the response.  :-)
